app/components/area-chart.gjs

```diff
@@ -0,0 +1,157 @@
+import Component from "@glimmer/component";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import didUpdate from "@ember/render-modifiers/modifiers/did-update";
+import { schedule } from "@ember/runloop";
+import Chart from "chart.js/auto";
+import { formatNumber } from "../helpers/format-number";
+
+export default class AreaChart extends Component {
+  limit = 8;
+  total = 0;
+  options = null;
+
+  scheduleChartRendering = (element) => {
+    schedule("afterRender", () => {
+      this.#renderChart(element && element.querySelector(".chart-canvas"));
+    });
+  };
+
+  resetChart = () => {
+    if (this.#chart) {
+      this.#chart.destroy();
+      this.#chart = null;
+    }
+  };
+
+  #chart;
```

Comment:
I usually advise against private syntax in components. With regular props - external things like parent and child components aren't able to access them either, and it's easier to debug stuff if it's not "hidden" 😃

Reply:
Done